public/components/wz-menu/wz-menu.js

```diff
@@ -851,11 +936,23 @@ class WzMenu extends Component {
       </button>
     );
 
+
+    const openSelectorsButton = (
+      <EuiButtonEmpty 
+        iconType="gear"
+        iconSide="right"
+        style={{ position: 'relative', right: 0 }}
+        onClick={()=> this.setState({ isSelectorsPopoverOpen: true })}
```

Comment:
Could be separate this on a new method and reutilize for `open` and `close` just denying the parameter 🤓